resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterSchedulerBackend.scala

```diff
@@ -322,7 +322,8 @@ private[spark] class KubernetesClusterSchedulerBackend(
             .addToRequests("cpu", executorCpuQuantity)
             .addToLimits("cpu", executorCpuQuantity)
           .endResources()
-          .withEnv(requiredEnv.asJava)
+          .addAllToEnv(requiredEnv.asJava)
+          .addToEnv(executorExtraClasspathEnv.toSeq: _*)
```

Comment:
was this not being used before?  I'm surprised checkstyle or something didn't alert that there was an unused identifier